dietpi/dietpi-software

```diff
@@ -3499,6 +3507,41 @@ _EOF_
 
 		fi
 
+		software_id=200 # DietPi Dashboard
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+
+			arch=
+			if (( $G_HW_ARCH == 10 )); then
+				arch='amd64'
+			elif (( $G_HW_ARCH == 3 )); then
+				arch='armv8'
+			elif (( $G_HW_ARCH == 2 )); then
+				arch='armv7'
+			else
+				arch='armv6'
+			fi
+
+			G_WHIP_MENU_ARRAY=(
+					'Stable' ': Use a release version of the DietPi Dashboard.'
+					'Nightly' ': Use an unstable version of the DietPi Dashboard. Could have bugs, but will probably have more features.'
+			)
+
+			if [[ $G_WHIP_RETURNED_VALUE == 'Stable' ]]
+			then
+				Download_Install "$(curl -sSfL 'https://api.github.com/repos/ravenclaw900/dietpi-dashboard/releases/latest' | mawk -F\" "/\"browser_download_url\": \".*dietpi-dashboard-$arch\"/{print \$4}")" /usr/local/bin/dietpi-dashboard
+			elif [[ $G_WHIP_RETURNED_VALUE == 'Nightly' ]]
+			then
+				Download_Install "https://nightly.link/ravenclaw900/DietPi-Dashboard/workflows/push-build/main/dietpi-dashboard-$arch.zip" dietpi-dashboard.zip
+				G_EXEC unzip -o dietpi-dashboard.zip
+				G_EXEC mv dietpi_dashboard /usr/local/bin/dietpi-dashboard
+			fi
+
+			G_EXEC chmod +x /usr/local/bin/dietpi-dashboard
```

Comment:
When not shipping a service, a location in PATH is fine for easy access. But we want to ship a systemd unit, right? Then I suggest to install the binary to `/opt/dietpi-dashboard/dietpi-dashboard`, where we may add other, like a readme, license, optional assets or such things, to have the option open.